internal/ccsmp/ccsmp_cache_session_prop_generated.go

```diff
@@ -0,0 +1,43 @@
+// pubsubplus-go-client
+//
+// Copyright 2021-2024 Solace Corporation. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ccsmp
+
+// Code generated by ccsmp_session_prop_generator.go via go generate. DO NOT EDIT.
+
+/*
+#include "solclient/solCache.h"
+*/
+import "C"
+
+const (
+	// SolClientCacheSessionPropCacheName: The identifier for the Distributed Cache to send cache requests to.
+	SolClientCacheSessionPropCacheName = C.SOLCLIENT_CACHESESSION_PROP_CACHE_NAME
+	// SolClientCacheSessionPropMaxMsgs: The maximum number of messages to retrieve from the cache for any one Topic.
+	SolClientCacheSessionPropMaxMsgs = C.SOLCLIENT_CACHESESSION_PROP_MAX_MSGS
+	// SolClientCacheSessionPropMaxAge: The oldest message (in seconds) to retrieve from the cache.
+	SolClientCacheSessionPropMaxAge = C.SOLCLIENT_CACHESESSION_PROP_MAX_AGE
+	// SolClientCacheSessionPropRequestreplyTimeoutMs: The timeout period (in milliseconds) to wait for a response from the cache. The minimum configurable value is 3000 (3 seconds). This is a protocol timer used internally by the API on each message exchange with solCache. A single call to solClient_cacheSession_sendCacheRequest() may lead to many request-reply exchanges with solCache and so is not bounded by this timer as long as each internal request is satisfied in time.
+	SolClientCacheSessionPropRequestreplyTimeoutMs = C.SOLCLIENT_CACHESESSION_PROP_REQUESTREPLY_TIMEOUT_MS
+	// SolClientCacheSessionPropReplyTo: Deprecated: The reply-to topic for the cache request. It is preferred to leave this property unspecified and let the API use the Session P2P Reply To Topic: ::SOLCLIENT_SESSION_PROP_P2PINBOX_IN_USE.
+	SolClientCacheSessionPropReplyTo = C.SOLCLIENT_CACHESESSION_PROP_REPLY_TO
+	// SolClientCacheSessionPropDefaultMaxMsgs: The default maximum number of message to retrieve from the cache for any one topic is 1.
+	SolClientCacheSessionPropDefaultMaxMsgs = C.SOLCLIENT_CACHESESSION_PROP_DEFAULT_MAX_MSGS
+	// SolClientCacheSessionPropDefaultMaxAge: Default maximum age is no maximum age (set to zero).
+	SolClientCacheSessionPropDefaultMaxAge = C.SOLCLIENT_CACHESESSION_PROP_DEFAULT_MAX_AGE
+	// SolClientCacheSessionPropDefaultRequestreplyTimeoutMs: The default timeout (in milliseconds) to wait for a response from the cache is 10000 (10 seconds).
+	SolClientCacheSessionPropDefaultRequestreplyTimeoutMs = C.SOLCLIENT_CACHESESSION_PROP_DEFAULT_REQUESTREPLY_TIMEOUT_MS
```

Comment:
EBP-33 outlines that the acceptance criteria should confirm these values are as expected.  Either thought an automated test or post your manual test results in the jira.

Reply:
If you've eyeballed them and printed them out for example I think that's okay on the manual side.  I wanted to make sure we'd at least confirmed that much